test_app/tests/authentication/authenticator_plugins/test_saml.py

```diff
@@ -212,8 +212,25 @@ def __init__(self):
     assert "mygroup" in rDict["Group"]
 
 
+def test_saml_create_via_api_without_callback_url(admin_api_client, saml_configuration):
+    del saml_configuration['CALLBACK_URL']
+
+    authenticator_data = {
+        "name": "Test SAML Authenticator",
+        "enabled": True,
+        "create_objects": True,
+        "remove_users": True,
+        "type": "ansible_base.authentication.authenticator_plugins.saml",
+        "configuration": saml_configuration,
+    }
+
+    url = get_relative_url("authenticator-list")
+    response = admin_api_client.post(url, data=authenticator_data, format="json", SERVER_NAME="dab.example.com")
+    assert response.status_code == 201, response.data
```

Comment:
Was this previously just a failure? Should we assert in here that the CALLBACK_URL is a fully qualified URL with a server we are expecting? No because we are testing it in test_get_fully_qualified_url is a fine answer for me.

Reply:
Done :+1: - and yes this failed previously :(